suite-native/react-native-graph/src/AnimatedLineGraph.tsx

```diff
@@ -355,12 +355,10 @@ export function AnimatedLineGraph<TEventPayload extends object>({
         (fingerX: number) => {
             const fingerXInRange = Math.max(fingerX - horizontalPadding, 0);
 
-            const lastDate = pointsInRange[pointsInRange.length - 1].date;
-
             // If there was graph error, points in range can be empty
-            if (!lastDate) {
-                return;
-            }
+            if (pointsInRange.length === 0) return;
```

Comment:
If you suggest this:

```tsx
const lastDate = pointsInRange[pointsInRange.length - 1]?.date;

if(!lastDate) return;
```
 
This version is little bit safer and future proof and will also pass https://www.totaltypescript.com/tips/make-accessing-objects-safer-by-enabling-nouncheckedindexedaccess-in-tsconfig (we sadly don't have this enabled)

Reply:
maybe you can try to turn it on for this package if it won't produce too many errors

Reply:
[6635ff5](https://github.com/trezor/trezor-suite/pull/11924/commits/6635ff5558831c8b861b8e5972d70e63efe107a9)